osu.Game.Rulesets.Osu/Difficulty/Evaluators/SpeedEvaluator.cs

```diff
@@ -24,7 +28,7 @@ public static class SpeedEvaluator
         /// <item><description>and how easily they can be cheesed.</description></item>
         /// </list>
         /// </summary>
-        public static double EvaluateDifficultyOf(DifficultyHitObject current)
+        public static double EvaluateDifficultyOf(DifficultyHitObject current, IReadOnlyList<Mod> mods)
```

Comment:
I think it would be better to replace this with `bool ignoreDistance` or `bool haveAutopilot` because it's not clear what you gonna do with mods list here